toolkit/tools/imagegen/installutils/installutils.go

```diff
@@ -288,8 +288,8 @@ func umount(path string) (err error) {
 	return
 }
 
-// PackageNamesFromSingleSystemConfig goes through the packageslist field in the systemconfig and extracts the list of packages
-// from each of the packagelists.
+// PackageNamesFromSingleSystemConfig goes through the packageslist and packages fields in the systemconfig, extracting
```

Comment:
Since we're here:
```suggestion
// PackageNamesFromSingleSystemConfig goes through the "PackageLists" and "Packages" fields in the "SystemConfig" object, extracting
```